jetty9-compat/src/main/docker/Dockerfile

```diff
@@ -12,7 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM jetty9:@@image.label@@
+FROM jetty9:${docker.label.prefix}${jetty.version}
+
+# Create env vars to identify image
+ENV GAE_IMAGE_NAME ${project.artifactId}
+ENV GAE_IMAGE_LABEL ${docker.label.prefix}8-jre
```

Comment:
The names/labels assigned to the images in the pom.xmls are:

- `openjdk8:${docker.label.prefix}8-jre`
- `jetty9:${docker.label.prefix}${jetty.version}`
- `jetty9-compat:${docker.label.prefix}${project.version}`

The images used by each docker file are:
- `gcr.io/google_appengine/base`
- `openjdk8:${docker.label.prefix}8-jre`
- `jetty9:${docker.label.prefix}${jetty.version}`

So I think they line up correctly.